lib/ruby2d/window.rb

```diff
@@ -236,6 +223,36 @@ def add_object(o)
         false
       end
     end
-    
+
+    def events_store(event_type)
```

Comment:
This sounds like a good idea, nice and clean. I'll update the PR later in the evening to do that.